### DISCUSSION TITLE
Coundown for react

### DISCUSSION THREAD

**Discussion by mamadreza-sheylani** [saadeghi/daisyui#899] [Q&A] [answered]
hi guys is there a way to use CountDown component in react
how to turn `style="--value:59;"` to  jsx?
------
I have tried rendering the value inside the span but the animation or the count down doesn't work.

**Comment by saadeghi** [comment, marked as answer]
JSX syntax of inline CSS would be something like:
```jsx
<span style={{"--value": 59}}>
```
